doc: Updating advanced_cluster docs to reflect new considerations for replicaset to sharded cluster transition (#2589)
mongodb/terraform-provider-mongodbatlas: docs/guides/advanced-cluster-new-sharding-schema.md

}
```

> Once the cluster type is adjusted accordingly, ensure to restart the application clients. If you don't reconnect the application clients, your application may suffer from data outages.

Reviewer: ```suggestion
Once the cluster type is adjusted accordingly, you must restart the application clients. If you don't reconnect the application clients, your application may suffer from data outages.
```
